client.go

```diff
@@ -280,6 +280,10 @@ func (c *Client) Delete(url, traceparent string) error {
 	if err != nil {
 		return err
 	}
+	if c.FiatUser != "" {
+	   request.Header.Set(SpinFiatUserHeader, c.FiatUser)
+	   request.Header.Set(SpinFiatAccountHeader, c.FiatUser)
```

Comment:
I've temporarily approved but TECHNICALLY account header isn't the right spot for the fiat user.  SHouldn't set that JUST the user header.

Reply:
ALSO be kinda nice if a test on this...

Reply:
tested in private environment and it's working fine